Backend/app/dashboard/provider-dashboard/src/API/Fleet/Registration.hs

```diff
@@ -71,4 +78,31 @@ fleetOwnerVerfiy req = withFlowHandlerAPI' $ do
   checkedMerchantId <- merchantCityAccessCheck merchantShortId merchantShortId req.city req.city
   void $ Client.callDynamicOfferDriverAppFleetApi checkedMerchantId req.city (.registration.fleetOwnerVerify) req
   token <- DR.generateToken person.id merchant.id req.city
+  unless (person.verified == Just True) $ QP.updatePersonVerifiedStatus person.id True
```

Comment:
What is the logic of verification, will it verify always?

Reply:
so this is for bpp dashboard side case , when the user is created using register api there phone no is not verified(In future we can include some other validation also ), once its no is verified (fleet/ verify api is called ) then will change this flag to true.